packages/components/src/alerts/Alert.tsx

```diff
@@ -0,0 +1,128 @@
+import { cn } from "@spin-spot/utils";
+
+interface AlertProps {
+  label: string;
+  type?: "alert" | "info" | "success" | "warning" | "error";
+  className?: string;
+  denyButtonLabel?: string;
+  acceptButtonLabel?: string;
+  onDeny?: () => void;
+  onAccept?: () => void;
```

Comment:
cambiaría esto por un prop "buttons" que te acepte cualquier cosa dentro, para que el usuario escoja los botones que quiera (no estrictamente accept y deny):
```suggestion
  buttons?: React.ReactNode
```